llm/prompt/template/openai/chat.py

```diff
@@ -38,3 +39,65 @@ def get_stylish_completion(prompt, style="Passionate and friendly"):
     messages = prompt_template.format_messages(style=style, prompt=prompt)
     response = chat(messages)
     return response.content
+
+
+def llmChain(prompt_input, chain_input):
+    llm = ChatOpenAI(temperature=0.9, model=llm_model)
+    prompt = ChatPromptTemplate.from_template(prompt_input)
+    chain = LLMChain(llm=llm, prompt=prompt)
+
+    return chain.run(chain_input)
+
+
+def simpleSequentialChain(prompt1, prompt2, chain_input):
+    llm = ChatOpenAI(temperature=0.9, model=llm_model)
+
+    first_prompt = ChatPromptTemplate.from_template(prompt1)
+    chain_one = LLMChain(llm=llm, prompt=first_prompt)
+
+    second_prompt = ChatPromptTemplate.from_template(prompt2)
+    chain_two = LLMChain(llm=llm, prompt=second_prompt)
+
+    overall_simple_chain = SimpleSequentialChain(
+        chains=[chain_one, chain_two], verbose=True
+    )
+
+    return overall_simple_chain.run(chain_input)
+
+
+def sequentialChain(prompt1, prompt2, prompt3, prompt4, review):
```

Comment:
maybe using **kwargs to make this function input with key and value. Keys like "summary", "reviews"... etc would better be passed from sample code(where this function be used). To make it more usable

Reply:
Great suggestion! I've reworked it using a list. Give it a look and tell me if it works for you. Thanks!